Lib/test/test_asyncio/test_subprocess.py

```diff
@@ -25,12 +25,26 @@
 if support.check_sanitizer(address=True):
     raise unittest.SkipTest("Exposes ASAN flakiness in GitHub CI")
 
+
+def get_quoted_sys_executable():
```

Comment:
Can you refactor the function and use shorter names for the attributes? it's a test file so it doesn't matter what we use. Or use an LRU-cache instead. OTOH, you can setup the test cases so that they have a test attribute (namely `cls.sysexec = ...` in `setUpClass` and you'd use as `self.sysexec` instead of `sys.executable` in the tests)

Reply:
Hi picnixz.
I think caching is overkill. I'd do it for my own code but not for opensource to keep it simple. I'll remove it.